snapcraft/plugins/_ros/wstool.py

```diff
@@ -59,6 +59,7 @@ def __init__(
         ros_package_path: str,
         wstool_path: str,
         ubuntu_sources: str,
+        ubuntu_keyrings: List[str],
```

Comment:
`List` should be reserved for return types, it is preferable to use `Sequence` here (https://docs.python.org/3.5/library/typing.html#typing.List)